batchflow/named_expr.py

```diff
@@ -401,6 +404,58 @@ def __repr__(self):
         return 'Unknown expression'
 
 
+class IF(NamedExpression):
+    """ Select either ``true`` or ``false``, based on ``condition``.
+    Useful for simple variables that change along the run of a pipeline.
+
+    Examples
+    --------
+    Select model mode based on the current pipeline iteration::
```

Comment:
Something is not OK with amount of symbols in the entire docstring: too much double `` and ::.